enterprise-suite/tests/e2e/scripts/wait_es_services.sh

```diff
@@ -11,6 +11,9 @@ CURL='curl --connect-timeout 1 --max-time 1 --output /dev/null --silent'
 ES_CONSOLE_URL=$(echo "$SERVICES" | grep expose-es-console | awk -F "|" '{ print $(NF-1)}')
 echo "ES-Console URL: ${ES_CONSOLE_URL}"
 
+# TODO remove before merging the pr
+minikube service --url expose-es-console
```

Comment:
Don't forget! :-)

Reply:
See it again. :-)

There is a way to get the URL without parsing the `service list` output. :-)